.github/workflows/close-stale.yml

```diff
@@ -20,3 +20,19 @@ jobs:
           exempt-issue-labels: never-stale
           exempt-pr-labels: never-stale
           any-of-labels: information-requested
+      - uses: actions/stale@v9
+        with:
+          days-before-issue-stale: 0
+          days-before-issue-close: 14
+          stale-issue-message: 'This package does not have an assigned owner and is considered unmaintained. As such this package is in feature-freeze and this issue will be closed with 14 days unless a new owner or a sponsor (@open-telemetry/javascript-approvers) for the feature is found. It is the responsibility of the author to find a sponsor for this feature.'
```

Comment:
do you want to add something for "have you consider becoming a code owner of this package?"

Reply:
Hmm, I think this is a good idea. Actually, we'd need some description of "how to become a component owner" which is currently missing. I'll open another PR for this as I think it's a separate topic that we should link to there.

Reply:
See #2287. Once merged, I'll add a link here so people know what to do to become a component owner.

Reply:
Great! Once you add the link, I think this Pr is good to get approved 😄

Reply:
Added the links (what unmaintained means, and how to become an owner): https://github.com/open-telemetry/opentelemetry-js-contrib/commit/cbfd085644df46bbd26fd70af2c02c4f2b3cce82 :slightly_smiling_face: